luxtronik/__init__.py

```diff
@@ -233,3 +272,34 @@ def _read_char(self):
         "Low-level helper to receive a signed int"
         reading = self._socket.recv(LUXTRONIK_SOCKET_READ_SIZE_CHAR)
         return struct.unpack(">b", reading)[0]
+
+
+class Luxtronik(LuxtronikData):
+    """
+    Wrapper around the data and the read/write interface.
+    Mainly to ensure backwards compatibility
+    of the read/write interface to other projects.
+    """
+
+    def __init__(self, host, port=LUXTRONIK_DEFAULT_PORT, safe=True):
+        super().__init__(safe=safe)
+        self.interface = LuxtronikSocketInterface(host, port)
+        self.read()
+
+    def read(self):
+        return self.interface.read(self)
+
+    def read_parameters(self):
+        return self.interface.read_parameters(self.parameters)
+
+    def read_calculations(self):
+        return self.interface.read_calculations(self.calculations)
+
+    def read_visibilities(self):
+        return self.interface.read_visibilities(self.visibilites)
+
+    def write(self, parameters=None):
+        if parameters is None:
+            self.interface.write(self.parameters)
+        else:
+            self.interface.write(parameters)
```

Comment:
A wrapper for `self.interface.write_and_read` would also be nice.

Reply:
Done